### ISSUE TITLE
npm ERR! notarget No matching version found for fs-extra@undefined

### ISSUE THREAD

**Issue by mishoboss** [mozilla-iot/gateway#906]
Anybody experiencing this on a clean install?
`npm ERR! notarget No matching version found for fs-extra@undefined`

Mac OS, latest node and npm.

**Comment by hobinjk** [comment]
It might be worthwhile to install and use `yarn` instead of `npm` by running `npm install -g yarn` or `brew install yarn`. You could then run `yarn` to install the dependencies. I'm not sure why fs-extra would be erroring like that as it usually just isn't installed since it's architecture dependent.

**Comment by mishoboss** [comment]
Using `yarn` is the same. Doesn't matter. I cleaned cache, updated node and npm, deleted node_modules folder, both global and local. Same. That's why I started to wonder if it may not be just me.

**Comment by dhylands** [comment]
Which platform are you installing onto?

**Comment by dhylands** [comment]
And what version of npm and node are you using?

**Comment by mishoboss** [comment]
I just cleaned everything possible I could imagine, reinstalled node, installed yarn globally, installed node modules, restarted the machine. Now works like a charm. Insanely strange issue, but important thing is it works! I close it.

**Comment by VadimGut** [comment]
Had the same issue for me it seemed to solve the problem to delete the node_modules and the lock file

**Comment by dejibimbolaAyo** [comment]
Having issues reinstalling strapi

`No matching version found for strapi-generate-admin@3.0.0-alpha.14.4`

**Comment by mrstegeman** [comment]
@dejibimbolaAyo How did you get this? `strapi-generate-admin` is not a dependency of the gateway at all.

**Comment by dejibimbolaAyo** [comment]
I just started out with strapi today. So reading the docs on [Admin panel](https://strapi.io/documentation/3.x.x/advanced/customize-admin.html)

So discovering I would need to add a `--dev` flag before creating a strapi project. So I deleted the test project i was working with and tried to create a new project with the `--dev` flag, but was getting the error. (i didn't keep the record of those errors though).

So I resorted to reinstalling strapi, and now faced with the error
`No matching version found for strapi-generate-admin@3.0.0-alpha.14.4`
